anagrams.rb

```diff
@@ -0,0 +1,30 @@
+def anagrams word
+  textfile =  File.new("enable.txt", "r")
+  dictionary_of_words = textfile.readlines
+  anagrams_of_word = []
+  dictionary_of_words.each_with_index do |item, index|
+    if (is_an_anagram_of?(word, item.chomp))
+      anagrams_of_word.push(item.chomp)
+    end
+  end
+  return anagrams_of_word
+end
+
```

Comment:
Thx for the tip about #readlines. This is a very readable method! It looks like you don't need to use #each_with_index since you never refer to the index. "#each do item" would have been sufficient.